Tests/Validator/ErrorElementTest.php

```diff
@@ -195,4 +195,18 @@ public function testFluidInterface()
         $this->assertSame($this->errorElement, $this->errorElement->addConstraint($constraint));
         $this->assertSame($this->errorElement, $this->errorElement->assertNotNull());
     }
+
+    public function testContextIsString()
```

Comment:
Please make the test title clearer to show we are testing exception throwing.

Like this one: https://github.com/sonata-project/SonataCoreBundle/pull/302/files#diff-6107bd6a6aa9c73dece5b23717e5fe8dR157